src/components/Button.js

```diff
@@ -3,18 +3,26 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name } = props;
+  const { name, color, twice } = props;
+  const styling = {
+    background: color,
+    width: twice ? '50%' : '25%',
+  };
   return (
-    <button type="button">
+    <button type="button" className="button" style={styling}>
       { name }
     </button>
   );
 };
 Button.propTypes = {
   name: PropTypes.string,
+  color: '#f5913e',
+  twice: false,
 };
```

Comment:
- This should contain the types of the respective props just like what you did for `name` prop, please do the same for `color` and `twice` props.